docs/quickstarts/insights-custom-repos/metadata.yml

```diff
@@ -0,0 +1,8 @@
+kind: QuickStarts # kind must always be "QuickStarts"
+name: insights-custom-repos
+tags: # If you want to use more granular filtering add tags to the quickstart
+  - kind: bundle
+    value: insights
+  - kind: application # use application tag for quickstart used by specific application
+    value: image builder
```

Comment:
This should match the application UI module ID. For image builder it is `imageBuilder`

Reply:
Thank you! I made the updates @Hyperkid123 !